style.css

```diff
@@ -53,6 +53,7 @@ a, a:hover, a:visited, a:active {
     border: 2px solid #bbb;
     padding: 0.4rem;
     font-size: 10pt;
+    margin:5px;
```

Comment:
This seems to negatively affect the positioning, for example the corners do not touch anymore
![grafik](https://user-images.githubusercontent.com/3027817/81202689-3fcec200-8fc7-11ea-88b5-569790fcb9e3.png)

Reply:
Oh 🤦 
Thanks for QA-ing this so well. 👍